### DISCUSSION TITLE
Old Packages

### DISCUSSION THREAD

**Discussion by CN1o2** [BEEmod/BEE2-items#4111] [Tech Support / Q&A]
With each version of _BEEMOD_, we also need to use the latest packages. 
However, is there a way to use old packages without some kind of bug getting in my way?

**Comment by vrad-exe** [comment]
It might work, but it's not supported. Why would you need to do that though?

**Reply by CN1o2** [reply]
I wish I could use 3d skyboxes. If not, is there a _UCP_'s for that?

**Reply by vrad-exe** [reply]
You'd need an old version of the compiler for that as well, the functionality was removed entirely. But they really weren't that good, the "bottomless pit" was basically just a pitch black hole. Literally just putting your map's goo at the bottom of a deep pit would look better.

We'd also prefer if you didn't use old versions since they're missing newer features and have a lot more bugs.
